packages/fgtclb/academic-persons/Classes/Provider/FrontendUserProvider.php

```diff
@@ -21,10 +22,10 @@ public function __construct(private readonly ConnectionPool $connectionPool) {}
     /**
      * @return array<int, array<string, mixed>>
      */
-    public function getUsersWithoutProfile(): array
+    public function getUsersWithoutProfile(?string $includePids = null, ?string $excludePids = null): array
```

Comment:
Should be `int[]` arrays directly and not the string list. And avoid dealing with null by enforcing array here. Empty arrays are not handled than.

Requires adopting handling code in the method block then.